inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/infer_request_config.cpp

```diff
@@ -51,20 +51,7 @@ namespace {
     };
 
     const std::vector<std::map<std::string, std::string>> AutoConfigs = {
-            {{InferenceEngine::KEY_AUTO_DEVICE_LIST , CommonTestUtils::DEVICE_CPU},
```

Comment:
again, maybe it was possible to leave the options, but implement a variant of the InferConfigTests that sets them directly to the plugin (via ie.SetConfig)

Reply:
reply the same above